src/osdc/Objecter.h

```diff
@@ -1724,6 +1724,8 @@ class Objecter : public md_config_obs_t, public Dispatcher {
   bool keep_balanced_budget = false;
   bool honor_pool_full = true;
 
+  std::atomic<int> extra_read_flags{0};
```

Comment:
There is already a global_ops_flags which is also atomic and appears only to be use by the filer Client::set_filer_flags and Client::get_filer_flags to set the same LOCALIZE_READS flag.

Why do we need two separate sets of additional flags, why can't this code just set the global_ops_flags?

Reply:
global_ops_flags is used for all ops, extra_read_flags is used only for read ops